Scripts/IncidentFields/IncidentFields.py

```diff
@@ -0,0 +1,77 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+
+def get_field_by_long_name(fields, name):
+    for field in fields:
+        if field['name'] == name:
+            return field
+    return None
+
+
+def get_field_by_short_name(fields, name):
+    for field in fields:
+        if field['cliName'] == name:
+            return field
+    return None
+
+
+def get_short_name(value):
+    return re.sub(r'[^A-Za-z0-9]', '', value).lower()
+
+
+def main():
+    # get incident fields
+    res = demisto.executeCommand('demisto-api-get', {'uri': '/incidentfields'})
+    if is_error(res):
+        return_error(res[0]['Contents'])
+
+    fields = res[0]['Contents']['response']
+
+    # 'fields' contains non-incident fields, as well, so let's make a version containing only incident fields
+    incident_fields = [field for field in fields if field['id'].startswith('incident_')]
+
+    # get arguments
+    args = demisto.args()
+
+    custom_only = False
+    if 'custom' in args and argToBoolean(args['custom']) is True:
+        custom_only = True
+
+    short_names = False
+    if 'short_names' in args and argToBoolean(args['short_names']) is True:
+        short_names = True
```

Comment:
Let's rename to something more readable.
Like use_cli_name, or use_short_name if that's the terminology the users know

Reply:
'cliName' is the property name that the `/incidentfields` API call gives to fields' shortnames, and I think is less descriptive, as what does it have to do with the CLI?

Not really sure what term is that users use (I don't think there is anything on the support site referencing it), but "short names" is what I call the shortened version of incident field names.  I'm happy to leave it as-is.